setup.py

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf8 -*-
+import codecs
+import re
+from os import path
+
+from setuptools import setup, find_packages
+
+
+def read(*parts):
+    file_path = path.join(path.dirname(__file__), *parts)
+    return codecs.open(file_path, encoding='utf-8').read()
+
+
+version = re.search(
+    r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
+    read('workflow/__init__.py'),
+    re.MULTILINE
+).group(1)
+
+setup(
+    name='workflow',
+    version=version,
+    description='',
+    long_description=read('readme.md'),
+    author='Håvard Wormdal Høiby',
+    author_email='',
+    license='MIT',
+    url='https://github.com/frigg/frigg-worker',
```

Comment:
I'm up for using frigg for CI, but lets keep this as github.com/havardh/workflow for now ;)